policyengine_canada/parameters/gov/provinces/nt/tax/income/credits/living_tax_credit/max_amount.yaml

```diff
@@ -0,0 +1,12 @@
+description: Northwest Territories allows for this maximum amount under the cost of living tax credit.
+values:
+  2022-12-31: 942 
```

Comment:
check 2022 amounts

Reply:
It seems that the max amount is 942 from 2005 - 2023: 
[https://www.fin.gov.nt.ca/sites/fin/files/resources/nwt_personal_income_tax_credits_english.pdf](https://www.fin.gov.nt.ca/sites/fin/files/resources/nwt_personal_income_tax_credits_english.pdf)